scripts/env.d/vars

```diff
@@ -13,3 +13,17 @@ case "$(uname)" in
 esac
 
 # vim: set et nu nobomb fenc=utf8 ft=bash ff=unix sw=2 ts=2:
+
+#----------------------------------------------------------------------------------------------------
+#    make sure system make is installed and reachable 
+#----------------------------------------------------------------------------------------------------
+#  find make
+MAKE=$(which make 2>/dev/null)
+
+if (( $? == 1 )); then 
+  echo " "
+  echo "ERROR: make needs to be installed to run devenv. "
+  echo " Install make using the command such as ' apt install build-essential' as root and try again. "
```

Comment:
I am not sure the leading and trailing spaces are meant to be. Besides, `' apt install build-essential'` has a leading space as well. Are they typo or on purpose?